linode_api4/groups/image.py

```diff
@@ -78,6 +79,7 @@ def create_upload(
         region: str,
         description: str = None,
         cloud_init: bool = False,
+        tags: List[str] = None,
```

Comment:
```suggestion
        tags: Optional[List[str]] = None,
```

Technically, a `List[str]` typed variable can't be a `None`. Wrapping it with `Optional` (can be imported from `typing`) will solve that.